src/mobile/apps/home/stylesheets/index.styl

```diff
@@ -4,6 +4,11 @@
 .home-page-section.main-side-margin
   margin-top 20px
 
+.home-page-section>h2
```

Comment:
I'm slightly disappointed that I had to add styles in force. @anandaroop What do you think about moving the `### Featured categories` heading into the component in Reaction? I like the semantics of not having it in the component, allowing the caller to name the rail whatever it wants...but the reality of having to style the same header in two different places in Force makes me question if leaving the heading off the Reaction component is more trouble than it's worth.

Reply:
This didn't jump out at me as a smell or anything, but I support moving the heading into the nav component if that feels right to you 👍